app/bundles/CampaignBundle/Tests/Functional/Fixtures/FixtureHelper.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\CampaignBundle\Tests\Functional\Fixtures;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Mautic\CampaignBundle\Entity\Campaign;
+use Mautic\CampaignBundle\Entity\Event;
+use Mautic\CampaignBundle\Entity\Lead as CampaignLead;
+use Mautic\LeadBundle\Entity\Lead;
+
+class FixtureHelper
+{
+    private EntityManagerInterface $em;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        $this->em = $em;
+    }
+
+    public function createContact(string $email): Lead
+    {
+        $contact = new Lead();
+        $contact->setEmail($email);
+
+        $this->em->persist($contact);
+
+        return $contact;
+    }
+
+    public function addContactToCampaign(Lead $contact, Campaign $campaign): CampaignLead
+    {
+        $ref = new CampaignLead();
+        $ref->setCampaign($campaign);
+        $ref->setLead($contact);
+        $ref->setDateAdded(new \DateTime());
+
+        $this->em->persist($ref);
+
+        return $ref;
+    }
+
+    public function createCampaign(string $name): Campaign
+    {
+        $campaign = new Campaign();
+        $campaign->setName($name);
+        $campaign->setIsPublished(true);
+
+        $this->em->persist($campaign);
+
+        return $campaign;
+    }
+
+    public function createCampaignWithScheduledEvent(Campaign $campaign, int $interval = 1, string $intervalUnit = 'd', \DateTime $hour = null): Event
```

Comment:
This method isn't creating a campaign. It should be named differently. For example
```suggestion
    public function addScheduledEventToCampaign(Campaign $campaign, int $interval = 1, string $intervalUnit = 'd', \DateTime $hour = null): Event
```